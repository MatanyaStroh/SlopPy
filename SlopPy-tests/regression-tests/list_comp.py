# make sure SlopPy works with list comprehensions

lst = ['1', '2', 'baaad', '3', '4']

xs = [int(line) for line in lst]

y = [20*x for x in xs]
assert y == [20, 40, 60, 80]

