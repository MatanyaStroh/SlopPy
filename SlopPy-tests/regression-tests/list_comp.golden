{NA <type 'exceptions.ValueError'> "invalid literal for int() with base 10: 'baaad'"
 Traceback (most recent call last):
  File "regression-tests/list_comp.py", line 5, in <module>
    xs = [int(line) for line in lst]

Locals: {'__builtins__': <module '__builtin__' (built-in)>, '__file__': 'regression-tests/list_comp.py', '__package__': None, '_[1]': [1, 2], 'lst': ['1', '2', 'baaad', '3', '4'], '__name__': '__main__', 'line': 'baaad', '__doc__': None}
}
{list.append(NA) | regression-tests/list_comp.py <module>:5}
